examples/src/omezarr/omezarr-viewer.tsx

```diff
@@ -44,30 +50,96 @@ export function OmezarrViewer({
     const canvas = useRef<HTMLCanvasElement>(null);
     const server = useContext(renderServerContext);
     const renderer = useRef<ReturnType<typeof buildAsyncOmezarrRenderer>>();
+    const imgRenderer = useRef<ReturnType<typeof buildImageRenderer>>();
+    const stash = useRef<StashedView>();
 
     // setup renderer and delete it when component goes away
     useEffect(() => {
         const c = canvas?.current;
         if (server?.regl && omezarr) {
             const numChannels = omezarr.colorChannels.length || 3;
-            renderer.current = buildAsyncOmezarrRenderer(server.regl, defaultDecoder, { numChannels });
+            renderer.current = buildAsyncOmezarrRenderer(server.regl, multithreadedDecoder, {
+                numChannels,
+                queueOptions: { maximumInflightAsyncTasks: 1 },
```

Comment:
I also exposed the queue control values as options when creating the renderer - I think the default value here was letting a great many requests hammer the S3 server, leading to slowness when rapidly canceling frames.